### ISSUE TITLE
NeoTRIS 4-Player Support

### ISSUE THREAD

**Issue by NeoArcade1** [finalburnneo/FBNeo#1834]
Cannot map buttons to the 3rd and 4th player to play NeoTRIS 4-Player Battle Mode

![NeoTRIS - Copy](https://github.com/user-attachments/assets/c219ad91-1fa7-471e-870f-aac9151dabfb)
![neotris1](https://github.com/user-attachments/assets/3a255df2-208e-4dab-9c5d-166ee1e2fa4c)

**Comment by barbudreadmon** [comment]
This game requires a special multitap for 3rd & 4th player. 
Afaik, this multitap is undocumented, so nobody knows how to emulate it.

**Comment by NeoArcade1** [comment]
It seems to be the same for Kizuna Encounter - Super Tag Battle
https://snk.fandom.com/wiki/Kizuna_Encounter:_Super_Tag_Battle_%224Way_Battle_Version%22

**Comment by barbudreadmon** [comment]
No, it's entirely different, Kizuna Encounter doesn't use that multitap, it uses a prototype mvs board.

**Comment by dinkc64** [comment]
@barbudreadmon any idea what's going on with minga12's sinden on the forum?  I think I have the proportional-mode inputs set almost exactly the same way as you did on your side, so I'm out of ideas :/  In the test video he recorded, the retical just flutters randomly along the screen when he aims at the center.

**Comment by barbudreadmon** [comment]
@dinkc64 could you send me that video ?

**Comment by dinkc64** [comment]
https://github.com/user-attachments/assets/529e9268-1c58-47cc-b0a3-0ec36bbf860f

It's above, note: it acts exactly how the mouse acts if you map it and use the positional "feature".  Is the sinden supposed to emulate a mouse?  I noticed he's got his mouse mapped to the gun x/y input.

**Comment by barbudreadmon** [comment]
My understanding is that (all of ?) those "lightgun devices" have 2 modes of operation, one where it sends on-screen coordinates, and one where it acts as a mouse.
It might be that this user is using the "mouse" mode of operation ?

**Comment by dinkc64** [comment]
Yes, I think it is using mouse from the looks of things, but, really confused why it's not working...
